src/shogun/multiclass/BruteKNNSolver.h

```diff
@@ -0,0 +1,65 @@
+/* This software is distributed under BSD 3-clause license (see LICENSE file).
+ *
+ * Copyright (c) 2012-2013 Sergey Lisitsyn
+ */
+
+#ifndef BRUTESOLVER_H__
+#define BRUTESOLVER_H__
+
+#include <shogun/lib/config.h>
+
+#include <shogun/lib/common.h>
+#include <shogun/multiclass/KNNSolver.h>
+#include <shogun/distance/Distance.h>
+
+namespace shogun
+{
+
+/**
+ * The BruteKNNSolver class is a solver class which inherit from KNNSolver class. It will use brute way
+ * to classify the object which mean compare the object against all training objects for each prediction.
+ */
```

Comment:
It will use brute way to classify the object which mean compare the object against all training objects for each prediction. -- should be

Test points are compared to all training data for each prediction.